docs/modules/ROOT/pages/explanations/decisions/postgres-monitoring.adoc

```diff
@@ -0,0 +1,51 @@
+= PostgreSQL by VSHN Availability check
+
+== Problem
+
+We need to monitor availability of PostgreSQL by VSHN, based on:
+
+* SLA: 99.99%
+* SLO: 99.995%
+* SLI: 99.995%
```

Comment:
A percentage for an SLI doesn't make sense. Also I wouldn't increase the SLO over 99.99%. Having an SLO that is higher than the SLA should allow us to react before we miss the SLA. For such a high SLA that's basically impossible anyway, so there is no point in having an even higher SLO